src/prg-e-f.asm

```diff
@@ -4398,7 +4399,7 @@ GetEnemyPointers:
 	CMP #$01
 	BNE loc_BANKF_F7A0 ; No, load the area as usual
 
-	LDY #$04 ; Yes; jars are always area #$04 for some reason
+	LDY #AreaIndex_Jar
```

Comment:
```suggestion
	LDY #AreaIndex_Jar ; Yes, load the area index used for common jars
```

Revising the comment instead of removing it, so that it's clear *why* it loads this area index number...

Not sure what we should use to differentiate "common jars" (the ones that always use `#$04`) versus the ones that are technically real sublevels. I'm not even truly sure what differentiates them other than the common ones likely store the entry point and return you there instead of actually warping you to a different area, but I can't really think of anything better.